### PR TITLE
Remove visibility change on global helper core service override.

### PR DESCRIPTION
**Pull request by xserna**
PR coming from https://github.com/ezsystems/LegacyBridge/pull/126#discussion_r353142084

### CODE REVIEW

**Comment by emodric** [comment]
Can you use `public: true` to mark it specifically as public, instead of relying on the default Symfony behaviour (and to avoid the deprecation message)?

You can also leave a comment why it is needed to be public.

**Comment by andrerom** [comment]
@emodric Maybe you can use suggest feature for what you had in mind?

File: bundle/Resources/config/templating.yml [hunk 1]
```diff
@@ -66,7 +66,6 @@ services:
     # Overriding core helper
     ezpublish.templating.global_helper:
         alias: ezpublish.templating.global_helper.legacy
```
Comment (emodric) [inline bundle/Resources/config/templating.yml]:
```suggestion
        alias: ezpublish.templating.global_helper.legacy
        # Needs to be public to use the service as a Twig global variable
        public: true
```

Comment (emodric) [reply bundle/Resources/config/templating.yml]: @andrerom Here you go. Care should be taken to not end up with duplicate `public` option, as I couldn't add the suggestion to a deleted line.

Comment (andrerom) [reply bundle/Resources/config/templating.yml]: @xserna Could you update PR with suggested change above?

Comment (xserna) [reply bundle/Resources/config/templating.yml]: > @xserna Could you update PR with suggested change above?

Sure, I've this on my todo list. Sorry for the delay, I'll update it now.

**Comment by andrerom** [comment]
Thanks @xserna 👏

**Pull request merged**